dashboard/test/ui/features/star_labs/contract_editor_sections.feature

```diff
@@ -1,4 +1,4 @@
-@as_student
+@as_student @no_phone
```

Comment:
This feature is use in /s/algebra, a very old lab which comprises 0.04% of our traffic (across all devices):

https://analytics.google.com/analytics/web/#/report/content-pages/a37745279w66217109p78966704/explorer-table.filter=algebra&explorer-table.plotKeys=%5B%5D

Didn't seem worth figuring out why this wasn't working on phone. FWIW it looked fine on my phone/Saucelabs.